ehr2meds/PREMEDS/preprocessing/extraction/extractor.py

```
@@ -0,0 +1,101 @@
+    """
+
+    def __init__(self, cfg) -> None:
+        self.cfg = cfg
+        self.test = cfg.test
+        logger.info(f"test mode = {self.test}")
+        self.input_path = cfg.paths.input
+        self.chunksize = cfg.data.chunksize
+        self.numeric_value = cfg.data.numeric_value
+        # Prepare parquet loader; will be used if extension is .parquet
+        path = dirname(self.input_path)
+        filename = split(self.input_path)[1]
```

Ruff (0.11.9)

Local variable `filename` is assigned to but never used

Remove assignment to unused variable `filename`

(F841)